python/moltres_xs.py

```diff
@@ -0,0 +1,269 @@
+"""
+     Python module that takes homogenized XS and prepares them for Moltres
+
+"""
+import json
+import io
```

Comment:
Judging from how you use "io", you probably don't actually need it. Wherever you use io.open, it looks like you could just use plain old open(), a built-in function in python.